mysqlx-connector-python 9.5.0

xhochy opened the pull request:
Fixes #14 

<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

xhochy committed:
Move to rattler

xhochy committed:
MNT: Re-rendered with conda-smithy 3.54.1 and conda-forge-pinning 2025.12.20.03.23.59

Other tools:
- conda-build 25.11.1
- rattler-build 0.54.0
- rattler-build-conda-compat 1.4.10

xhochy committed:
Add osx-arm64

xhochy committed:
MNT: Re-rendered with conda-smithy 3.54.1 and conda-forge-pinning 2025.12.20.03.23.59

Other tools:
- conda-build 25.11.1
- rattler-build 0.54.0
- rattler-build-conda-compat 1.4.10

xhochy committed:
Migrate for abslq2

xhochy committed:
MNT: Re-rendered with conda-smithy 3.54.1 and conda-forge-pinning 2025.12.20.07.05.32

Other tools:
- conda-build 25.11.1
- rattler-build 0.54.0
- rattler-build-conda-compat 1.4.10

xhochy committed:
Patch for new abseil

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

xhochy committed:
test: native

xhochy committed:
MNT: Re-rendered with conda-smithy 3.54.1 and conda-forge-pinning 2025.12.20.07.05.32

Other tools:
- conda-build 25.11.1
- rattler-build 0.54.0
- rattler-build-conda-compat 1.4.10

xhochy committed:
Clean patch rebase

xhochy commented:
@conda-forge/mysqlx-connector-python This is ready for review. Migrates abseil & friends, updates the version, moves to v1 and enables osx-arm64.
